src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -139,86 +128,87 @@ export class EntitySelectComponent<
 
   @Input() additionalFilter: (e: E) => boolean = (_) => true;
 
-  ngOnChanges(changes: SimpleChanges): void {
-    if (changes["form"]) {
-      this.form.valueChanges.subscribe((value) => {
-        this.updateAvailableOptions().then((_) => {});
-      });
-    }
-  }
-
-  private async loadAvailableEntities() {
-    this.loading.next(true);
-
-    const entities = [];
-    for (const type of this._entityType) {
-      entities.push(...(await this.entityMapperService.loadType<E>(type)));
-    }
-    this.allEntities = entities
-      .filter((e) => this.additionalFilter(e))
-      .sort((a, b) => a.toString().localeCompare(b.toString()));
-
-    await this.updateAvailableOptions();
-
-    this.loading.next(false);
-  }
-
-  private async updateAvailableOptions() {
-    const includeInactive = (entity: E) =>
-      this.includeInactive || entity.isActive;
-    const includeSelected = (entity: E) =>
-      asArray(this.form.value).includes(entity.getId());
-
-    const newAvailableEntities = this.allEntities.filter(
-      (e) => includeInactive(e) || includeSelected(e),
-    );
+  private entitiesForType: Resource<E[]> = resource({
```

Comment:
I first thought this is only loading the entities of a single type (of possibly multiple types). Maybe naming it `allEntities` again or `entitiesOfTypes` could be a bit more clear in that regard?